deck.gl/examples/pure-js/angular/carto3/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "deckgl-for-angular",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~11.2.0",
+    "@angular/cdk": "^11.2.0",
+    "@angular/common": "~11.2.0",
+    "@angular/compiler": "~11.2.0",
+    "@angular/core": "~11.2.0",
+    "@angular/forms": "~11.2.0",
+    "@angular/material": "^11.2.0",
+    "@angular/platform-browser": "~11.2.0",
+    "@angular/platform-browser-dynamic": "~11.2.0",
+    "@angular/router": "~11.2.0",
+    "@deck.gl/carto": "8.5.0",
+    "@deck.gl/core": "8.5.0",
+    "@deck.gl/layers": "8.5.0",
+    "@ngrx/store": "^11.0.1",
+    "@turf/bbox-polygon": "^6.3.0",
+    "@turf/boolean-intersects": "^6.3.0",
+    "@types/mapbox-gl": "^2.1.0",
+    "deck.gl": "8.5.0",
```

Comment:
and here

Reply:
Updated to 8.6.5. Sometimes there are breaking changes in minor versions, so I prefer to pin the example to a specific version.